lib/services/ElevationChart.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:charts_flutter/flutter.dart' as charts;
+import 'package:flutter/material.dart';
+import 'package:hiking4nerds/services/route.dart';
+
+import 'osmdata.dart';
+
+class ElevationChart extends StatelessWidget {
+  final List<charts.Series> seriesList;
+  final bool interactive;
+  final bool withLabels;
+
+  int index;
+
+  ElevationChart(this.seriesList, {this.interactive, this.withLabels});
+
+  /// Creates a [LineChart] with sample data and no transition.
+  factory ElevationChart.withData(HikingRoute route,
```

Comment:
the factory is one way, but i think the more intuitve way would be to set the route via the constructor (@required) and use the route data inside build to create the chart correctly